core-api/src/main/java/com/redhat/lightblue/Response.java

```diff
@@ -68,6 +86,10 @@ public Response(JsonNodeFactory jsonNodeFactory) {
         this.jsonNodeFactory = jsonNodeFactory;
     }
 
+    public String getHostname() {
```

Comment:
We're not doing this now, but there's nothing preventing using the lightblue core classes on the client side. If that happens, Response will contain the hostname of the client, not the server.

Insead of this: use a mutable hostname string with getter/setters. Parse/tostring it as a regular field. Initialize if to HOSTNAME in ctor.